static/js/components/forms/MenuItemForm.test.tsx

```diff
@@ -53,106 +53,39 @@ describe("MenuItemForm", () => {
-        internalLink: activeItem.targetContentId,
-      })
+      contentLink: "",
     })
   })
 
-  it("has radio buttons to switch between external and internal links", () => {
-    const setFieldValueStub = jest.fn()
-    const wrapper = renderInnerForm(
-      {},
-      {
-        values: {
-          menuItemTitle: "",
-          menuItemType: LinkType.Internal,
-          externalLink: "",
-          internalLink: "",
-        },
-        setFieldValue: setFieldValueStub,
-      },
-    )
-    const itemTypeBtns = wrapper.find('input[name="menuItemType"]')
-    expect(itemTypeBtns).toHaveLength(2)
-    const internalBtn = itemTypeBtns.at(0)
-    const externalBtn = itemTypeBtns.at(1)
-    expect(internalBtn.prop("checked")).toBe(true)
-    expect(internalBtn.prop("value")).toEqual(LinkType.Internal)
-    expect(externalBtn.prop("checked")).toBe(false)
-    expect(externalBtn.prop("value")).toEqual(LinkType.External)
-    // @ts-expect-error Not going to simulate the event
-    externalBtn.prop("onChange")()
-    expect(setFieldValueStub).toHaveBeenCalledWith(
-      "menuItemType",
-      LinkType.External,
-    )
-    // @ts-expect-error Not going to simulate the event
-    internalBtn.prop("onChange")()
-    expect(setFieldValueStub).toHaveBeenCalledWith(
-      "menuItemType",
-      LinkType.Internal,
-    )
+  it("renders with the correct initial values if given an active item with link", () => {
+    const activeItem = {
+      text: "text",
+      targetContentId: "content-id",
+    }
+    const wrapper = renderForm({
+      activeItem,
+    })
+    expect(wrapper.prop("initialValues")).toEqual({
+      menuItemTitle: activeItem.text,
+      contentLink: activeItem.targetContentId,
+    })
   })
 
-  it("renders an internal link dropdown if the 'internal' radio is selected", () => {
+  it("renders a link dropdown", () => {
     const wrapper = renderInnerForm(
       {},
       {
         values: {
           menuItemTitle: "",
-          menuItemType: LinkType.Internal,
-          externalLink: "",
-          internalLink: "",
+          contentLink: "",
         },
       },
     )
-    const relationField = wrapper.find('RelationField[name="internalLink"]')
+    const relationField = wrapper.find('RelationField[name="contentLink"]')
     expect(relationField.exists()).toBe(true)
-    expect(wrapper.find('Field[name="externalLink"]').exists()).toBe(false)
-  })
-
-  it("renders an external link dropdown if the 'external' radio is selected", () => {
-    const wrapper = renderInnerForm(
-      {},
-      {
-        values: {
-          menuItemTitle: "",
-          menuItemType: LinkType.External,
-          externalLink: "",
-          internalLink: "",
-        },
-      },
-    )
-    const extLinkField = wrapper.find('Field[name="externalLink"]')
-    expect(extLinkField.exists()).toBe(true)
-    expect(wrapper.find('RelationField[name="internalLink"]').exists()).toBe(
-      false,
-    )
```

Comment:
Awesome. Good job getting rid of stale tests as well. This will simplify our codebase.